packages/amazonq/src/app/chat/activation.ts

```diff
@@ -18,7 +18,7 @@ export async function activate(context: ExtensionContext) {
         void amazonq.LspController.instance.trySetupLsp(context, {
             startUrl: AuthUtil.instance.startUrl,
             maxIndexSize: CodeWhispererSettings.instance.getMaxIndexSize(),
-            isVectorIndexEnabled: CodeWhispererSettings.instance.isLocalIndexEnabled(),
+            isVectorIndexEnabled: false,
```

Comment:
I know we had a discussion on this in the internal team channel, but can you pull out in this PR what vector indexing is, what features it enables, etc. Would be super useful just in case we have to re-reference it